modules/react/tooltip/lib/TooltipContainer.tsx

```diff
@@ -45,23 +34,47 @@ const defaultTransformOrigin = {
   horizontal: 'center',
 } as const;
 
-export const TooltipContainer = styled('div')<TooltipContainerProps>(
-  {
-    ...type.levels.subtext.medium,
+const tooltipTranslateVars = createVars('positionX', 'positionY');
+
+/**
+ * Keyframe for the Tooltip animation.
+ */
+const tooltipAnimation = keyframes({
+  '0%': {
+    opacity: 0,
+    transform: `translate(${tooltipTranslateVars.positionX}, ${tooltipTranslateVars.positionY})`,
+  },
+  '100%': {
+    opacity: 1,
+    transform: 'translate(0)',
+  },
+});
+
+export const tooltipContainerStencil = createStencil({
+  vars: {
+    tooltipTransformOriginHorizontal: '',
+    tooltipTransformOriginVertical: '',
+  },
+  base: ({tooltipTransformOriginHorizontal, tooltipTransformOriginVertical}) => ({
+    ...system.type.subtext.medium,
     display: 'inline-flex',
     position: 'relative',
-    padding: space.xs,
-    color: colors.frenchVanilla100,
+    padding: system.space.x3,
+    color: base.frenchVanilla100,
```

Comment:
my guess is this is the text color?

Reply:
Yeah. I updated this. I had to look at the spec.